src/python/pants/option/options_bootstrapper.py

```diff
@@ -121,6 +119,7 @@ def create(
           absolute paths. Production use-cases should pass True to allow options values to make the
           decision of whether to respect pantsrc files.
         """
+        args = tuple(args)
```

Comment:
To ensure we can fingerprint them (previously we inadvertently relied on CliAlias tupling them for us.